.circleci/config.yml

```diff
@@ -176,21 +176,6 @@ step_run_all_tests: &step_run_all_tests
         " >> .rspec-local
 
         bundle exec rake ci
-step_release_docs: &step_release_docs
-  run:
-    name: Upload release docs
-    command: S3_DIR=trace bundle exec rake release:docs
```

Comment:
Can you also delete the respective rake task `release:docs`?

Reply:
Done